spec/src/main/asciidoc/license-efsl.adoc

```diff
@@ -25,8 +25,8 @@ document, or portions thereof, that you use:
 * link or URL to the original Eclipse Foundation document.
 * All existing copyright notices, or if one does not exist, a notice
   (hypertext is preferred, but a textual representation is permitted)
-  of the form: "Copyright (c) [$date-of-document]
-  Eclipse Foundation, Inc. \<<url to this license>>"
```

Comment:
I don't know if this substitution is required. It's meant as an example. By filling in the date and the specific url, then this might be used verbatim and it might not be accurate. At least by leaving the "substitution text" users would know what is expected. That's how I have been reading this section.

Reply:
Ah, ok, but it does seem like the EFSL license URL should be there?

Reply:
This is the generic form that has been used for all of the skeletal Specs as well as the Platform, Web Profile, and Managed Beans Specs.  So, if it's decided that the url should be filled out explicitly, then all of the instances of this file would also need to be updated.  I'd tend towards going for consistency than "correctness" at this point...